docker-run/githistory.log

```diff
@@ -0,0 +1,76 @@
+*   11425ee : Merge remote-tracking branch 'pcgeek86/main' into main
```

Comment:
This file can be removed I guess.

Reply:
This file is there to keep the changes that are made, before moved to our repo without any history. I have no clue how we handle source we copy and modify - only link and LICENSE copy?